src/libraries/System.Private.CoreLib/src/System/Threading/NativeRuntimeEventSource.PortableThreadPool.cs

```diff
@@ -362,5 +363,31 @@ public unsafe void ThreadPoolWorkingThreadCount(uint Count, ushort ClrInstanceID
             data[1].Reserved = 0;
             WriteEventCore(60, 2, data);
         }
+
+#if !ES_BUILD_STANDALONE
+        [System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:UnrecognizedReflectionPattern",
+                   Justification = EventSourceSuppressMessage)]
+#endif
+        [Event(65, Level = EventLevel.Verbose, Message = Messages.IO, Task = Tasks.ThreadPool, Opcode = Opcodes.IOPack, Version = 0, Keywords = Keywords.ThreadingKeyword)]
+        private unsafe void ThreadPoolIOPack(
+            IntPtr NativeOverlapped,
+            IntPtr Overlapped,
+            ushort ClrInstanceID = DefaultClrInstanceId)
+        {
+            if (!IsEnabled(EventLevel.Verbose, Keywords.ThreadingKeyword))
```

Comment:
Just a nit, it may be unnecessary to check this here. When used, I figure this method would look similar to the private `ThreadPoolIODequeue` event method, where it is supported by a public helper method that would take the managed `Overlapped` object and pass in the hash code to this event method. Then it would be better to check `IsEnabled` in the public helper method before getting the hash code. Same comment for the other file.